app/src/main/res/values/strings.xml

```diff
@@ -13,20 +13,16 @@
     <string name="content_description_private_browsing_button">Enable private browsing</string>
     <!-- Content description (not visible, for screen readers etc.): "Private Browsing" menu button. -->
     <string name="content_description_disable_private_browsing_button">Disable private browsing</string>
-    <!-- Explanation for how sessions work -->
-    <string name="sessions_intro_description">Sessions help you return to the sites you visit often and those you\'ve
```

Comment:
Removing this without removing references to it in the code is causing the build to break. If we want to remove this description string, we also must remove the code where it's referred to.